internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -96,7 +96,12 @@ CloudFrontDistribution:
             {{- end}}
       {{- if .CDNConfig.Static}}
         - Id: !Sub 'copilot-${AppName}-${EnvironmentName}-s3'
+        {{- if .CDNConfig.Static.ImportedBucket}}
           DomainName: {{.CDNConfig.Static.ImportedBucket}}
+        {{- end}}
+        {{- if .CDNConfig.Static.StaticSiteBucket}}
```

Comment:
can they only have a single static site service exposed via cloudfront? or can we support multiple origins?

Reply:
not for this release. we can make it a union type for the env manifest ad hoc